src/robot/running/librarykeywordrunner.py

```diff
@@ -214,7 +218,7 @@ def _get_run_kws_keywords(self, given_args):
     def _get_run_kws_calls(self, given_args):
         if 'AND' not in given_args:
             for kw_call in given_args:
-                yield [kw_call,]
+                yield [kw_call, ]
```

Comment:
I don't think a space is needed here and there are some other style changes that I don't agree with. It would be better if this PR just added the feature and didn't touch style elsewhere.